vars/sconsBuild.groovy

```diff
@@ -134,7 +134,7 @@ def call(Map config = [:]) {
     if (config['clean']) {
         clean_files = config['clean']
     }
-    clean_files += ' install build {daos_m,iof,cart-Linux}.conf'
+    clean_files += ' install build {daos_m,daos,iof,cart-Linux}.conf'
```

Comment:
FYI, just a bit of shell-geek-ery:

```suggestion
    clean_files += ' install build {daos{,_m},iof,cart-Linux}.conf'
```

:grin:

Reply:
To be clear, not at all suggesting you make this change.  Was totally just a FYI.

Reply:
thanks.  I think I'll pass.  We don't need daos_m for very long anyway